### ISSUE TITLE
Status page: Sync now CTA

### ISSUE THREAD

**Issue by nataliekwong** [airbytehq/airbyte#27894]
We see in Fullstory that users who continue to use the Job History page are doing so for two reasons:

- They want to trigger a manual sync
- They want to "watch" the job run. This could be because they 1) want to know it's starting and 2) that it has completed, so they can go to their destination to use the data. (Edit: to be tackled here: https://github.com/airbytehq/airbyte/issues/28159)

As we consider how the status page will evolve, we can make it more useful by: 
- "Sync x enabled streams" should just be "sync now" and mirror the button on the job history page in color, to make it obvious you can do the same.

A few clarification questions (now answered)
1. Is there a reason we went with the wording "x enabled streams"? I imagine this is useful when differentiating between syncing all enabled streams or just syncing a single stream.  However, syncing a single stream is not possible in the grey dot options today.
Answer: original design
3. (Related to the first question) Is the ability to sync a single stream planned?
Answer: no, not currently.

Scope of this issue: 
1. Change the CTA text to "Sync now"
2. Change the button color to primary to mimic the job history page

**Comment by nataliekwong** [comment]
> Another suggestion raised was to use a progress bar on the status page so that people can "see" it's moving. While it would be difficult to fully implement this (would require several inputs, like # records read, previous throughput of the connection), one suggestion here would be to focus on showing the # of records read, so that users have security that the sync is running and don't need to babysit any problems.

I'd like to split off another issue around what a progress bar enhancement could look like, but have a clarifying question on this: we currently emit in the logs how many records are read. @chandlerprall could you help me understand how we might be able to utilize that information in the FE for a pared down version of a progress bar? Would that be on the Move team or more of a FE project?

**Comment by chandlerprall** [comment]
> Is there a reason we went with the wording "x enabled streams"? I imagine this is useful when differentiating between syncing all enabled streams or just syncing a single stream. However, syncing a single stream is not possible in the grey dot options today.

Agreed. My belief is the "x enabled streams" wording was added to the design to make the button's resulting action explicit compared to (not-currently-existing) ability to sync individual stream(s).

> (Related to the first question) Is the ability to sync a single stream planned?

It is listed in the stream centric PRD as a nice to have at this phase; to my knowledge it is not included in any planning.

> could you help me understand how we might be able to utilize that information in the FE for a pared down version of a progress bar? Would that be on the Move team or more of a FE project?

If anything exists only in the logs we would need backend to collect & expose that data. However, from the jobs response we're already receiving on the status page we know `recordsEmitted, bytesEmitted, recordsCommitted` per attempt, rolled up into one total but also per stream. I printed these to the screen and kicked off a few syncs:

* Github: moves slowly, bytes/record data is made available slowly; doesn't seem like a set cadence but it wouldn't make for a fun progress viewing
* Postgres: similarly produces the stats slowly, isn't a fun progress to watch; I have connection where each stream finishes but took a couple minutes for the job itself to populate with the stats
* E2E: appears to emit stats at the end

I would rely on the move team's guidance for data expectations for the jobs endpoint, and go from there. Doesn't appear to me that the data+cadence available is enough to power user-facing progress.
